FOR_RELEASE/GameData/UmbraSpaceIndustries/Akita/Parts/Akita_Cab.cfg

```
@@ -50,7 +50,7 @@ PART
 		lightBrightenSpeed = 2.5
 		lightDimSpeed = 2.5
 		resourceAmount = 0.04
-		animationName = AkitaLights
+		animationName = #LOC_USI_Animation_AkitaLights//AkitaLights
```

```animationName``` is an internal variable.  It shouldn't be localized.
```suggestion
		animationName = AkitaLights
```